src/app.rs

```diff
@@ -229,6 +230,10 @@ fn default_numcpu() -> u32 {
     num_cpus::get() as u32
 }
 
+fn default_build_repo_base() -> PathBuf {
+    PathBuf::from("build-repo")
```

Comment:
Whoops, this needs to be a full path and a default isn't clear. I'll drop this commit.